chart/compass/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -233,3 +255,6 @@ spec:
       - name: gardener-kubeconfig
         secret:
           secretName: {{ .Values.gardener.secretName }}
+      - name: audit-log-script
+        configMap:
+          name: {{ .Values.audit-logs.scriptName }}
```

Comment:
Why that variable now is not global and has a hyphen in the name?

Reply:
fixed.